topics/persistence.md

```diff
@@ -60,6 +66,8 @@ for faster restarts, and in the event of bugs in the AOF engine.
 
 Note: for all these reasons we'll likely end up unifying AOF and RDB into a single persistence model in the future (long term plan).
 
+Since Redis 7.0.0, when `aof-use-rdb-preamble` is set to yes, the BASE file uses RDB format.
```

Comment:
how is this related to this section?
i think we can choose not to document that option at all.
or if we document it, we should mention the version where it was first introduced, and what it did before 7 too

Reply:
Pay attention to the above sentence: ”for all these reasons we'll likely end up unifying AOF and RDB into a single persistence model in the future (long term plan).“ i think when `aof-use-rdb-preamble` is set to yes, we can do that, right ?

Reply:
i don't think that's it.
the section is titled `Ok, so what should I use`, and it mentions the potential for data loss, vs the overheads of AOF (constant writes, not during AOFRW)
it discusses the difference between a periodic snapshots (e.g. fork every hour), vs AOF (write every command), which has a rewrite that could be much more sparse..

it is still a long term plan for us to unify these (i.e. you can define both, and you'll end up with only AOF, but it's rewrite will be periodic).
i don't think this PR changes that.

we may choose to mention preamble somewhere (if there's a line that mentions that AOFRW is being inefficient since it generates text file), but then we'll mention the version that introduced preamble, not 7.0.
i don't think 7.0 adds a major change with regards to this section.

Reply:
removed